source/common/stats/grpc_metrics_service_impl.h

```diff
@@ -117,16 +117,22 @@ class MetricsServiceSink : public Sink {
 
   void flushCounter(const Counter& counter, uint64_t) override {
     io::prometheus::client::MetricFamily* metrics_family = message_.add_envoy_metrics();
+    io::prometheus::client::MetricType counter_type = io::prometheus::client::MetricType::COUNTER;
+    metrics_family->set_type(counter_type);
```

Comment:
mild preference for doing this inline here and below, e.g.
metrics_family->set_type(io::prometheus::client::MetricType::COUNTER)